alicloud/resource_alicloud_cen_child_instance_attachment.go

```diff
@@ -0,0 +1,169 @@
+package alicloud
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/cbn"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAlicloudCenChildInstanceAttachment() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAlicloudCenChildInstanceAttachmentCreate,
+		Read:   resourceAlicloudCenChildInstanceAttachmentRead,
+		Delete: resourceAlicloudCenChildInstanceAttachmentDelete,
+
+		Schema: map[string]*schema.Schema{
+			"cen_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"child_instance_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"child_instance_type": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
+					value := v.(string)
+					if value != "VPC" && value != "VBR" {
+						errors = append(errors, fmt.Errorf("%s must be one of VPC and VBR", k))
+					}
+
+					return
+				},
+			},
+			"child_instance_region_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
```

Comment:
I think `child_instance_region_id` should be Optional. If it is not set, you can get a default from provider.